Userland/Utilities/rm.cpp

```diff
@@ -37,6 +39,13 @@ int main(int argc, char** argv)
 
     bool had_errors = false;
     for (auto& path : paths) {
+        if (!no_preserve_root) {
+            if (path == "/") {
```

Comment:
nitpick, but in this case the outer if only contains the inner, so there's no point to the 2 separate ifs (aka, `&&`)

Reply:
Modified and I didn't put a `and` haha